README.md

```diff
@@ -6,13 +6,38 @@
 
 ![ORAS](https://github.com/oras-project/oras-www/raw/main/docs/assets/images/oras.png)
 
+## Project status
+
+### v1: stable
+
+As there are various stable projects depending on the ORAS Go library, the
+[`v1`](https://github.com/oras-project/oras-go/tree/v1) branch
+is maintained for API stability, dependency updates, and security patches.
+All `v1.*` releases are based upon this branch.
```

Comment:
can we clarify which branch is “this”

Reply:
I believe this is clear from the previous sentence.

Reply:
To be clear, I meant since this is markdown the newline isn't registered, the last sentence appears in the same paragraph as the previous sentence where the `v1` branch name and link was noted.